xarray/core/combine.py

```diff
@@ -605,21 +560,141 @@ def auto_combine(datasets, concat_dim=_CONCAT_DIM_DEFAULT,
     --------
     concat
     Dataset.merge
-    """  # noqa
-
-    # Coerce 1D input into ND to maintain backwards-compatible API until API
-    # for N-D combine decided
-    # (see https://github.com/pydata/xarray/pull/2553/#issuecomment-445892746)
-    if concat_dim is None or concat_dim == _CONCAT_DIM_DEFAULT:
-        concat_dims = concat_dim
-    elif not isinstance(concat_dim, list):
-        concat_dims = [concat_dim]
+    """
+
+    if concat_dim is '_not_supplied':
+        concat_dim = _CONCAT_DIM_DEFAULT
+    else:
+        message = """In xarray version 0.14 `auto_combine` will be deprecated,
```

Comment:
This message will get formatted in a weird way, due to the extra indentation.  A nice way to handle multi-line strings like these while keeping the code formatted well is to use `textwrap.dedent()` from Python's standard library.